RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc

```diff
@@ -346,6 +399,82 @@ void MkFitGeometryESProducer::addTECGeometry(mkfit::TrackerInfo &trk_info) {
   edm::LogVerbatim("MkFitGeometryESProducer") << ostr.str();
 }
 
+void MkFitGeometryESProducer::findRZBox(const GlobalPoint &gp, float &rmin, float &rmax, float &zmin, float &zmax) {
+  float r = gp.perp(), z = gp.z();
```

Comment:
```suggestion
  float absr = std::fabs(gp.perp()), absz = std::fabs(gp.z());
```

Reply:
`r` is already non-negative

Reply:
Ok, then please remove the `std::fabs` operating on `r`

Reply:
review addressed